.github/workflows/push-trigger.yml

```
@@ -1,30 +1,64 @@
       - 1.*
       - develop
       - MOSIP*
       - release*
       - ES-842
-
+  pull_request:
+    types: [opened, reopened, synchronize]
+  workflow_dispatch:
+    inputs:
+        message:
+        description: 'Message for manually triggering'
```

actionlint (1.7.9)

input settings of workflow_dispatch event is scalar node but mapping node is expected

(syntax-check)